src/app/components/featured-articles/featured-articles.component.ts

```diff
@@ -9,6 +13,49 @@ import { CarouselComponent } from '../../shared/carousel/carousel.component';
   templateUrl: './featured-articles.component.html',
   styleUrl: './featured-articles.component.scss'
 })
-export class FeaturedArticlesComponent {
+export class FeaturedArticlesComponent implements OnInit, OnChanges, OnDestroy {
   @Input() topicName: string = '';
+  autoplayInterval: number = 3000;
+  responsiveOptions?: CarouselResponsiveOptions[];
+  articles$: Observable<ArticleItems[] | []> = new Observable();
```

Comment:
No point in having | [] syntax, it can always be an empty array where you have an array. Same below